app/src/main/res/values-cs/strings.xml

```diff
@@ -0,0 +1,441 @@
+  <br /><br />
+  Finance poslané na tuto adresu se zobrazí po jednom potvrzení.
+  <br /><br />
+  Poplatek <b>%1$s%%</b> bude aplikován.
+  ]]></string>
+
+  <!-- //////////////// receive with open channel //////////////// -->
+
+  <string name="receive_with_open_title">Příchozí platba</string>
+  <string name="receive_with_open_help_desc">Co to je?</string>
+  <string name="receive_with_open_help">Pro přijetí Lightning platby musíte mít dostatečnou příchozí likviditu.\n\nPokud žádnou nemáte, Phoenix automaticky vytvoří kanál dle potřeby. Zřizovací poplatek pokryje cenu vytvoření kanálu a přidání likvidity do něj.</string>
+  <string name="receive_with_open_amount_label">Máte nevyřízenou příchozí platbu</string>
+  <string name="receive_with_open_cost"><![CDATA[Přijmutí této platby vyžaduje sestavovací poplatek <b>%1$s</b> (%2$s).]]></string>
+  <string name="receive_with_open_accept">Přijmout (vyprší za %1$ss)</string>
+  <string name="receive_with_open_decline">Odmítnout</string>
+  <string name="receive_with_open_expired">Tato platba vypršela!</string>
+
+  <!-- //////////////// send //////////////// -->
+
+  <string name="send_init_paste">Vložit platební požadavek</string>
+  <string name="send_init_reading">Čtu sken…</string>
+
+  <string name="send_balance_prefix">Zůstatek:</string>
+  <string name="send_use_all_balance">Poslat vše</string>
+  <string name="send_loading">Kontroluji platbu…</string>
+  <string name="send_invalid_pr_generic">Tato platba není platná. Musel jste naskenovat neplatný požadavek.\n\nProsím zkuste to znovu.</string>
+  <string name="send_amount_error">Prosím, vložte platné množství.</string>
+  <string name="send_amount_error_balance">Množství je vyšší než váš zůstatek.</string>
+  <string name="send_amount_error_swap_out_too_small">Minimálně 10000 satosthi za on-chain platbu.</string>
+  <string name="send_destination_label">Odeslat na</string>
+  <string name="send_description_label">Popis</string>
+  <string name="send_error_sending">Nastala chyba při odesílání platby. Žádné peníze nebyly odeslány.</string>
+  <string name="send_pay_button">Zaplatit</string>
+  <string name="send_swap_in_progress">Probíhá směna…</string>
+  <string name="send_swap_error">Směna selhala. Prosím zkuste to znovu.</string>
+  <string name="send_swap_required_desc">Toto je bitcoinová adresa, platba bude vytvořena směnou.</string>
+  <string name="send_swap_required_button">Požadavek na směnu</string>
+  <string name="send_swap_complete_recap_amount">Množství</string>
+  <string name="send_swap_complete_recap_fee">Poplatek pro težaře</string>
+  <string name="send_swap_complete_recap_total">Celkem</string>
+  <string name="send_swap_complete_cannot_afford">Celkové množství převyšuje váš zůstatek.\n\nSnižte množství, nebo to zkuste později až se sníží poplatky.</string>
+
+  <!-- //////////////// scan //////////////// -->
+
+  <string name="scan_request_camera_access">Umožněte přístup ke kameře</string>
```

Comment:
Povolte přístup ke kameře
Povolit přístup ke kameře

Not sure with the context.

Reply:
It's a button, so the latter will do.